### PR TITLE
Integration of DFS-Perf and Alluxio

### PR DESCRIPTION
**Pull request by pfxuan**
This is a work-in-progress port from Tachyon to Alluxio. This PR works on most of benchmark workloads.

### CODE REVIEW

**Comment by pfxuan** [comment]
@swatDong, this PR is ready for review.

File: src/main/java/pasalab/dfs/perf/benchmark/iterate/IterateThread.java [hunk 1]
```diff
@@ -56,7 +56,6 @@ protected void syncBarrier(int iteration) throws IOException {
     }
   }
 
-  @Override
```
Comment (swatDong) [inline src/main/java/pasalab/dfs/perf/benchmark/iterate/IterateThread.java]: Thanks for fixing this.
Would you mind fixing other "Override" in other benchmarks?

Comment (pfxuan) [inline src/main/java/pasalab/dfs/perf/benchmark/iterate/IterateThread.java]: Sure! Will fix them in next commit.

File: src/main/java/pasalab/dfs/perf/fs/PerfFileSystemAlluxioFS.java [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  @Override
+  public void close() throws IOException {
+  }
+
+  @Override
+  public void connect() throws IOException {
+    ClientContext.getConf().set(Constants.MASTER_HOSTNAME, mFilePath.getHost());
+    ClientContext.getConf().set(Constants.MASTER_RPC_PORT,
+        Integer.toString(mFilePath.getPort()));
+    ClientContext.init();
```
Comment (swatDong) [inline src/main/java/pasalab/dfs/perf/fs/PerfFileSystemAlluxioFS.java]: I think it's better to put this step in the constructor. And the `mFilePath` field can be removed.

**Comment by swatDong** [comment]
@pfxuan Really thank you for this patch! Left few comments. Other parts LGTM.

**Comment by pfxuan** [comment]
@swatDong, all your comments have been addressed. Please help to check. Thank you for your support!

**Pull request merged**